api.py

```diff
@@ -0,0 +1,23 @@
+def bookings_options_get(message):
```

Comment:
I suggest to move this file to something like `mock_server/api.py`

Reply:
Done.